packaging/tests/tas-email/pom.xml

```diff
@@ -36,11 +36,22 @@
             <artifactId>jakarta.mail-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>${dependency.testng.version}</version>
+            <scope>test</scope>
+        </dependency>
         <dependency>
             <groupId>com.epam.reportportal</groupId>
             <artifactId>agent-java-testng</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why is this needed only here? 🤔

Reply:
Rest of the tas- modules already got those dependency. it's set in a global pom DependencyManagement, so we need those dependencies also in a child poms to actually use it